docker-compose.yml

```diff
@@ -11,15 +10,17 @@ services:
     <<: *redash-service
     command: server
     ports:
-      - "5000:5000"
+      - "5001:5000"
```

Comment:
why do we need change the port to 5001?

Reply:
I had something else running locally on port 5000 - ive adjusted the PR accordingly back to port 5000